internal/config/configuration.go

```diff
@@ -8,6 +8,14 @@ import (
 )
 
 type Configuration struct {
+
+	Server  ServerConfiguration
+	Mongodb MongodbConfiguration
+	Redis   RedisConfiguration
+	S3      S3StorageConfiguration
+	Python  MicroserviceConfiguration
+	Sentry  SentryConfiguration
+=======
```

Comment:
remove the left over markers na
also cant you see that we you have double Server,Mongdb and redis?